Hyper/PlayerData/InventorySystem/Items/Item.cs

```diff
@@ -9,13 +9,15 @@ internal abstract class Item
 
     public abstract bool IsStackable { get; }
 
-    public virtual CursorType Cursor => CursorType.Crosshair;
-
     public virtual void Use(Scene scene) { }
 
     public virtual void SecondaryUse(Scene scene) { }
 
     public virtual void Use(Scene scene, IChunkWorker chunkWorker, float time) { }
 
     public virtual void SecondaryUse(Scene scene, IChunkWorker chunkWorker, float time) { }
+
+    public virtual void Up() { }
```

Comment:
`Up` and `Down` are only used in `Pickaxe`, maybe we could keep them only there?

Reply:
I don't like it. I think calling it looks cleaner if we con't have to check is the item in hand is a pickaxe. Also its not the only method used only in hammer so I think its better to keep things consistant